secure/vars.sample.php

```diff
@@ -5,14 +5,19 @@
 manually upload to your server, outside of git.
  */
 
-$ENVIRONMENT = 'dev';
-$web_url = 'http://localhost/';
-
+$ENVIRONMENT = System::ENVIRONMENT_DEV; // Use only ENVIRONMENT_DEV or ENVIRONMENT_PROD
+$ENABLE_DEV_ONLY_FEATURES = System::ENABLE_DEV_ONLY_FEATURES; // Boolean value
+$LOCAL_HOST_CONNECTION = System::LOCAL_HOST; // Boolean value, set to false for running on websites
+$REGISTER_OPEN = true;
 $SC_OPEN = true;
-$register_open = true;
+$USE_NEW_BATTLES = false;
+$WAR_ENABLED = true;
+$REQUIRE_USER_VERIFICATION = false;
+
+$WEB_URL = 'http://localhost/';
 
 //Host needs to match the DB's host. For docker setup, rename it to "shinobi-chronicles-db".
-$host = 'localhost';
-$username = 'database_username';
-$password = 'database_password';
-$database = 'database_name';
+$HOST = 'localhost';
+$USERNAME = 'database_username';
+$PASSWORD = 'database_password';
+$DATABASE = 'database_name';
```

Comment:
Will this break the existing usages of `System` if we merge as is?

Reply:
Yes, right now things are in a bit of a broken state. I'm currently working through some bugs with SysV2. I could revert some of these changes if we're wanting to push out just the index changes. I'd think I'll have things running smoothly within a few hours of getting home tonight.

Reply:
Can wait for you to have this ready to go, just making sure this is something we're fixing 👍

Reply:
Will have it fixed 💯 I'll likely have some additional variables that will need added on dev & production.